tests/phpunit/unit/Extensions/AssetsProviderTest.php

```diff
@@ -188,7 +188,7 @@ class AssetsProviderTest extends AbstractExtensionsUnitTest
     protected function getApp($boot = true)
     {
         $app = parent::getApp(false);
-        $mock = $this->getMock('\Bolt\Asset\BoltVersionStrategy', ['getVersion'], [$app['filesystem']->getFilesystem('extensions'), $app['asset.salt']]);
+        $mock = $this->getMock('\Bolt\Asset\BoltVersionStrategy', ['getVersion'], [$app['filesystem']->getFilesystem('extensions')->getDir(''), $app['asset.salt']]);
```

Comment:
Did you do a "Gawain" and forget that we're on PHP 5.5+ now :stuck_out_tongue_winking_eye: 

Zero stress & changes needed here btw… just woke up in a very good, and very cheeky, mood :wink: